src/openloco/objects/objectmgr.h

```diff
@@ -45,6 +45,13 @@ namespace openloco
         competitors,
         scenario_text,
     };
+#pragma pack(push,1)
+
+    struct land_object {
```

Comment:
Already introduced in 9c1e8461bc6cc1982b1b30fe837377f2da55edbe on master:

https://github.com/OpenLoco/OpenLoco/blob/ddf564f447fc416262715b0f856011489850d6bc/src/openloco/objects/land_object.h#L8-L13